x/simplelru/lru.go

```diff
@@ -115,7 +127,7 @@ func (m *LRU[K, T]) Get(key K) (T, time.Time, bool) {
 	return zero, time.Time{}, false
 }
 
-// prune removes entries if space is needed. It tries
+// Prune removes entries if space is needed. It tries
```

Comment:
comment is exported, function is not....

Reply:
@karasz eh? you explicitly complained about this comment change when I unexported prune

Reply:
@karasz commit droped